src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/beanvalidation/BeanValidationQuickFix.java

```diff
@@ -81,7 +81,7 @@ public CodeAction resolveCodeAction(JavaCodeActionResolveContext context) {
     private void removeConstraintAnnotationsCodeActions(Diagnostic diagnostic, JavaCodeActionContext context, List<CodeAction> codeActions) {
 
         final String annotationName = diagnostic.getData().toString().replace("\"", "");
-        final String name = Messages.getMessage("RemoveConstraintAnnotation", annotationName);
+        final String name = Messages.getMessage("RemoveConstraintAnnotation", annotationName.substring(annotationName.lastIndexOf('.') + 1));
```

Comment:
Rather than directly applying substring(), it will be better to get the simple name by using the function like below.

protected static String getSimpleName(String fqName) {
        int idx = fqName.lastIndexOf('.');
        if (idx != -1 && idx != fqName.length() - 1) {
            return fqName.substring(idx + 1);
        }
        return fqName;
    }

Reply:
I have changed it accordingly. Thanks